File: 2828/2828.js
// Runtime
// Details
// 75ms
// Beats 33.90%of users with JavaScript
// Memory
// Details
// 45.21MB
// Beats 16.83%of users with JavaScript


var isAcronym = function(words,  s) {
 var isAcronym = true;

  if (words.length != s.length) {
    return false;
  } else {
    for (var i = 0; i < words.length; i++) {
      if (words[i][0] != s[i]) {
        return false;
      }
    }
  }
  return isAcronym;
}


var words = ["alice","bob","charlie"]; var s = "abc";

console.log(isAcronym(words, s));

File: 2828/2828.dart
// Accepted
// Editorial
// Solution
// Runtime
// Details
// 404ms
// Beats 6.67%of users with Dart
// Memory
// Details
// 147.84MB
// Beats 93.33%of users with Dart

bool isAcronym(List<String> words, String s) {
  bool isAcronym = true;

  if (words.length != s.length) {
    return false;
  } else {
    for (int i = 0; i < words.length; i++) {
      if (words[i][0] != s[i]) {
        return false;
      }
    }
  }
  return isAcronym;
}

main() {
  var words = ["never", "gonna", "give", "up", "on", "you"];
  var s = "ngguoy";
  print(isAcronym(words, s));
  return 0;
}
